cmd/gdsutil/main.go

```diff
@@ -53,26 +55,29 @@ func main() {
 	app.Name = "gdsutil"
 	app.Version = pkg.Version()
 	app.Usage = "utilities for operating the GDS service and database"
-	app.Commands = []cli.Command{
+	app.Commands = []*cli.Command{
 		{
 			Name:     "ldb:keys",
 			Usage:    "list the keys currently in the leveldb store",
 			Category: "leveldb",
 			Action:   ldbKeys,
 			Before:   openLevelDB,
 			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:   "d, db",
-					Usage:  "dsn to connect to trisa directory storage",
-					EnvVar: "GDS_DATABASE_URL",
+				&cli.StringFlag{
+					Name:    "d",
+					Aliases: []string{"db"},
```

Comment:
I went the opposite direction with `Name: "db"` and `Aliases: []string{"d"}` - is it better to do it this way?